src/dosext/mfs/mfs.c

```diff
@@ -360,6 +361,18 @@ static int cds_drive(cds_t cds)
     return -1;
 }
 
+static uint16_t GetDataSegmentInDOS(void)
```

Comment:
Yes, this function, as per most of the int2f/12xx, can only be called with DOS stack. If we are InDOS we have the DOS stack already set up for use. Do you want me to add a comment to the source?

Reply:
```
--------D-2F1203-----------------------------
INT 2F U - DOS 3.0+ internal - GET DOS DATA SEGMENT
        AX = 1203h
Return: DS = data segment of IBMDOS.COM/MSDOS.SYS
Note:   for DOS prior to version 5.0, the data segment is the same as the code
          segment
```

> Yes, this function, as per most of the int2f/12xx, can only be called with DOS stack.

But RBIL doesn't say that, so why do you think so?

Reply:
Thinking about it, there's probably no reason for DOS to need the stack within it. So many 12xx functions do need it, I convinced myself that it did. I'll rename the function, and remove the comment.

Reply:
And this is quite bad actually, because if it
is not the InDOS one, then there is no guarantee
it can be safely called while InDOS. :)
But I suppose there is no other way of supporting
dr-dos, so lets merge this.